### PR TITLE
fix(#3950): picker style issue

### PR DESCRIPTION
**Pull request by ashika01**
_Issue #, if available:_ #3950 

_Description of changes:_ Just a line of styling that got lost over history.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ashika01** [comment]
There was no real way of testing this because the change lies more on styling side of things. Any ideas welcome.

**Comment by sammartinez** [comment]
> There was no real way of testing this because the change lies more on styling side of things. Any ideas welcome.

Can you provide a screenshot of the change? With and without the style change

File: packages/aws-amplify-react/src/Widget/Picker.tsx [hunk 1]
```diff
@@ -66,10 +66,15 @@ export default class Picker extends Component<IPickerProps, {}> {
 		const accept = this.props.accept || '*/*';
 
 		const theme = this.props.theme || AmplifyTheme;
+		const pickerStyle = Object.assign(
+			{},
+			{ position: 'relative' },
```
Comment (jordanranz) [inline packages/aws-amplify-react/src/Widget/Picker.tsx]: We should move this to a css module file so it can be shared between frameworks.

Comment (ashika01) [reply packages/aws-amplify-react/src/Widget/Picker.tsx]: I didn't know we are using cssm. I just returned it to its old state. I will go add and do that.

Comment (jordanranz) [reply packages/aws-amplify-react/src/Widget/Picker.tsx]: Spoke offline, I think we will consolidate this style into our new Stencil component and skip doing the work to move this to a CSS module for now.

**Pull request merged**